WordPress/gutenberg
# [RNMobile] Update `Podfile.lock` file of Gutenberg demo app

## Description


https://github.com/WordPress/gutenberg/pull/34886 updated the pod references used in `react-native-editor` which are also used in the Gutenberg demo app, however, we forgot to update the `Podfile.lock` file, for this reason, this PR updates that file.

## How has this been tested?



- Run command `npm ci`.
- Run command `npm run native ios`.
- Observe that the `Podfile.lock` file is not modified.

## Screenshots 
N/A

## Types of changes




Bug fix

## Checklist:
